pages/data-design/avo-tracking-plan/events.mdx

```diff
@@ -4,14 +4,21 @@ import { Callout } from 'nextra/components'
 
 *Learn about the events in your tracking plan*
 
-An event represents something that a user does or that happens in response to their actions. For example "Signup Attempted" is an action that the user did, while "Credit Card Validated" is something that happened to the user.
+An event represents a user action (for example: "Signup Started") or something that happens in response to a user action (for example "Credit Card Validated).
+
+In Avo, your events live under the Events section of the Tracking plan and can be organized into categories.
+
+## Creating events
+
+New events can be created from the Events section by clicking the "New Event" button at the top of the view. They can also be created in context from a [Metric](/data-design/avo-tracking-plan/metrics) or a [Property](/data-design/avo-tracking-plan/metrics).
```

Comment:
Add the command pallette here? And the quicklink into the creation by pressing _c_ from anywhere within the app?